Sources/AppcuesKit/Data/Analytics/AnalyticsTracker.swift

```diff
@@ -28,9 +28,18 @@ internal class AnalyticsTracker: AnalyticsTracking, AnalyticsSubscribing {
 
     // maintain a batch of asynchronous events that are waiting to be flushed to network
     private let syncQueue = DispatchQueue(label: "appcues-analytics")
+
+    // this is the background analytics processing queue - 10 sec batch
     private var pendingActivity: [Activity] = []
```

Comment:
I didn't rename things for simpler review in this commit, but I do think I'll make another commit pass to rename `pendingActivity` to `backgroundActivity`, and `flushWorkItem` to `backgroundFlushWorkItem` - and whatever other naming clarifications should be done. I think it would ideally be more clear why there are two code paths and exactly which case they are for.